ci/scripts/utils/githubpr.py

```diff
@@ -3,8 +3,11 @@
 import os
 import re
 
-from github import Github, GithubException, InputFileContent, UnknownObjectException
-from wxflow import which
+from github import Auth, Github, GithubException, InputFileContent, UnknownObjectException
+from wxflow import which, Logger
+
+# Initialize logger with environment variable for logging level
+logger = Logger(level=os.environ.get("LOGGING_LEVEL", "DEBUG"), colored_log=False)
```

Comment:
Just checking, should this remain at `LOGGING_LEVEL=DEBUG`?  Or was that just for testing?

Also, I have been looking for a way to disable colors when logging.  Thanks for showing off this `colored_log=False`!

Reply:
David, All of the python scripts that use the **wxflow** Logger are set at the DEBUG level, so I thought it best to follow form here.  Notice that we can alter the Level at runtime with the environment variable LOGGING_LEVEL, so it appears the intent was to have DEBUG as the default.